### ISSUE TITLE
command-a  in url box is mistaken for control-a, and opens all the sleeping tabs in a window

### ISSUE THREAD

**Issue by jpbianchi** [deanoemcke/thegreatsuspender#847]
Please complete the following information when submitting a feature request or bug report.
 * Extension version: v7.0.109
 * Browser name & version: Chrome 71.0.3578.98
 * Operating system & version: OSX 10.13.6

It's been some time now that, when I want to select a whole url by clicking in the url box and do the usual command-a, Chrome does not, as it should, select the text at all but unfreezes all the frozen tabs in that window, as if I had typed control-a (which is my setting to do that).  
It's as if the extension confuses command and control when the cursor is in the url box.  
This is new, I never had this problem for a long time so I don't know what changed.  
It's not a huge problem, very annoying, and when chrome re-opens 60 sleeping pages at once, my 12 cores get used up to 85% for quite some time.  If I type ctrl-s to freeze all the tabs to try to stop the problem, nothing happens, the extension continues re-opening all the tabs.

**Comment by deanoemcke** [comment]
@jpbianchi For the CPU load issue. It sounds like this is happening when you accidentally unsuspend all your tabs? If this is the case, it's really a chrome issue, there's not much I can do about it unfortunately (the CPU load I mean). 

I'll investigate whether I can somehow interrupt the tab loading and force the tabs to suspend quickly.

**Comment by jpbianchi** [comment]
Yes, it happens when I accidentally unsuspend all the tabs.  I know it's
Chrome reloading the tabs, but it's your app telling Chrome which tabs to
reopen right?
So maybe telling Chrome in a timely manner would help, which also would
allow you to interrupt it when the user hits ctrl-S.
Thx for your help and consideration

**Comment by deanoemcke** [comment]
I have just committed a change that allows tabs to be suspended even during loading. This should help somewhat when trying to stop all your tabs from reloading (v7.0.165).

It seems that you are also suggesting that I stagger the unsuspending process when unsuspending a large number of tabs at the same time? This is a little off topic for this issue. Would you mind creating a new issue for this (as a feature request)? 

Thanks!

**Comment by jpbianchi** [comment]
Hey, thx for making that change so quickly.  I'll test it.
Staggering the unsuspending lots of tabs was just a suggestion, but a
useful one I think since crazy chrome almost freezes the computer for 2
minutes, so I'll be happy to request that feature.
Thx again.
